src/sentry/models/appconnectbuilds.py

```diff
@@ -0,0 +1,54 @@
+"""Database models to keep track of the App Store Connect builds for a project.
+
+If a project enables the App Store Connect source to download dSYMs directly from Apple we
+need to keep track of which builds have already been downloaded.
+"""
+
+from django.db import models
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class AppConnectBuild(Model):
+    """A single build that exists or has existed on App Store Connect.
+
+    Builds on App Store Connect expire, so they do disappear from the API.  This table does
+    not get pruned however, once we know of a build we know of it forever.
+
+    An individual build can be identified by the tuple of (bundle_id, platform,
+    bundle_short_version, bundle_version).
+    """
+
+    __core__ = True
```

Comment:
This is probably `core = False`. Core is badly named, but it's just used for import/exporting data in on-prem

Reply:
Could you expand on this?  I basically went with the related table is `__core__ = True` so copied that.  If we set this to false the data will not be exported and after an import this table would be empty?

This would work for here, the next task would re-populate the table and re-download the dSYMs.  In fact if the DIFs are not copied along with an export (they're in filestore).  Actually looking at the `FileBlob` table that is `__core__ = False` so maybe if I understand correctly this forces us to be `__core__ = False` as well.

If you could let me know if my understanding is correct that'd be great.  I'll also split the migration off to its own PR now.

Reply:
Yeah, basically it means that if someone is using this in on-prem, and they want to migrate to Saas, then any rows in this table will not be included in the migration over.

Basically, anything related to config should be `__core__ = True`. So orgs/projects/members/settings etc. Anything related to events/groups/files/releases shouldn't, since they can end up quite large. In practice, since `__core__` is named unclearly, people have applied it kind of haphazardly, and it probably needs an audit at some point.

Reply:
As a very general comment, I did briefly look for a guide on develop.sentry.dev about adding new tables and creating migrations but didn't find one.  There's a bunch of things I learned stumbling through this that could be made a smoother experience with such a guide 😉